### ISSUE TITLE
Data Licensing Question

### ISSUE THREAD

**Issue by jtarbox** [BSData/catalogue-development#98]
I'm considering making a free website (w/ minimal ads to pay for hosting) for Warhammer 40k to be able to quickly search and filter units by keywords, features, and wargear, with some baseline data showing and maybe a bit "mathhammer" as well.  I would want to keep this updated using the data generated by your team for 40k, but would like permission to use this data before I attempt any such endeavor.   The entirety of it is still just thoughts in my head, so I have nothing concrete to demo in this regard.  In the past I've done a character analyzer for the Everquest 2 MMO, but the data and math in this case is a much less guesswork.

**Comment by amis92** [comment]
I've come up with a draft of what we might do, without any legal advice. This is a draft:

## Licensor

**BSData** is the [BSData GitHub organization](https://github.com/BSData).

## License

[![Creative Commons Attribution 4.0 International License](https://i.creativecommons.org/l/by/4.0/88x31.png)](http://creativecommons.org/licenses/by/4.0/)

BSData datafiles are licensed under [Creative Commons Attribution 4.0 International License](http://creativecommons.org/licenses/by/4.0/) by BSData contributors with the **Notice** below.

## Notice

BSData has copyright on all datafiles which are considered databases, with the exception of data that contains publication references, including but not limited to names, costs, rules, profiles and their included characteristics. Those are considered citations of the published material which the publication reference describes, and BSData cites these publications without explicit permission, unless otherwise noted. References are provided on best-effort basis. Elements with no publication reference are considered to possibly be taken from any of the publications defined in the datafile, its parent datafile, or in other places of the repository.

TL;DR BSData has copyright on datafiles (as databases) but most often not their content/data, and we don't license any part to which we don't have rights to.

**Comment by DrTobogganMD** [comment]
This issue has had no activity for 8 months and can safely be closed.
